workflows/sarek.nf

```diff
@@ -552,29 +567,33 @@ workflow SAREK {
 
         // Gather used softwares versions
         versions = versions.mix(CONVERT_FASTQ_INPUT.out.versions)
-        versions = versions.mix(FASTQ_ALIGN_BWAMEM_MEM2_DRAGMAP.out.versions)
+        versions = versions.mix(FASTQ_ALIGN_BWAMEM_MEM2_DRAGMAP_SENTIEON.out.versions)
     }
 
     if (params.step in ['mapping', 'markduplicates']) {
 
         // ch_cram_no_markduplicates_restart = Channel.empty()
         cram_markduplicates_no_spark = Channel.empty()
+        cram_sentieon_dedup = Channel.empty()
         cram_markduplicates_spark    = Channel.empty()
 
         // STEP 2: markduplicates (+QC) + convert to CRAM
 
-        // ch_bam_for_markduplicates will countain bam mapped with FASTQ_ALIGN_BWAMEM_MEM2_DRAGMAP when step is mapping
+        // ch_bam_for_markduplicates will contain bam mapped with FASTQ_ALIGN_BWAMEM_MEM2_DRAGMAP_SENTIEON when step is mapping
         // Or bams that are specified in the samplesheet.csv when step is prepare_recalibration
         cram_for_markduplicates = params.step == 'mapping' ? bam_mapped : input_sample.map{ meta, input, index -> [ meta, input ] }
-
         // if no MD is done, then run QC on mapped & converted CRAM files
         // or the input BAM (+converted) or CRAM files
         cram_skip_markduplicates = Channel.empty()
 
         // Should it be possible to restart from converted crams?
         // For now, conversion from bam to cram is only done when skipping markduplicates
 
-        if (params.skip_tools && params.skip_tools.split(',').contains('markduplicates')) {
+        if (
+            params.skip_tools &&
+            params.skip_tools.split(',').contains('markduplicates') &&
+            !(params.tools && params.tools.split(',').contains('sentieon_dedup'))
```

Comment:
don't think I agree with this style of alignment :D we need a linter so we won't argue about it 😆

Reply:
Let me know which alignment style you like, and we can fix this particular one 😊